### ISSUE TITLE
Read more on/off

### ISSUE THREAD

**Issue by ocean123** [giansi/grav-plugin-blog#5]
Hi,

Is there on option to make it possible to turn off/on the "Continue Reading..." button?
Maybe something in the header of blog.md   readmore: true/false

best regards

**Comment by giansi** [comment]
Why would you like to disable that button?

**Comment by ocean123** [comment]
yes, in some blog views we don't want the read more option because the article has to be showed completly.

**Comment by ocean123** [comment]
for example for reviews. that are short messages.

**Comment by ocean123** [comment]
http://syndes.nl/review
